### DISCUSSION TITLE
the download for connectors is so slowly!

### DISCUSSION THREAD

**Discussion by laglangyue** [apache/pulsar#19638] [Q&A] [answered]
as the title，is there someway to solve the question，or we can down the connector by maven repository. I find some jar of connector in maven repository，but the jar file can't  work for pulsar  connectors

**Comment by laglangyue** [comment]
![image](https://user-images.githubusercontent.com/35491928/221421141-cb6864d0-8a3b-484a-ab0b-7d0cbae9dd79.png)
such as the canal connector, I download it in my ali cloud ,the speed of it is so slowly

**Comment by tisonkun** [comment, marked as answer]
This has nothing to do with the Pulsar community. It's mainly about general network issues.
